### PR TITLE
Add self.prerequisite_view for step()

### PR DESCRIPTION
**Pull request by mfalesni**

### CODE REVIEW

File: airgun/navigation.py [hunk 1]
```diff
@@ -45,6 +45,10 @@ def am_i_here(self, *args, **kwargs):
 
     def go(self, _tries=0, *args, **kwargs):
         super(BaseNavigator, self).go(_tries=_tries, *args, **kwargs)
+        try:
+            self.prerequisite_view = self.parent
```
Comment (abalakh) [inline airgun/navigation.py:49]: Is there any functionality tied to `self.prerequisite_view`? Or its only reason is to have a nice shortcut without need of using hasattr/try-except?

Comment (abalakh) [reply airgun/navigation.py:49]: Asking as i personally like `self.parent` more, as it's already there with no properties duplication and it's basically shorter. If there's no functional meaning we could keep using it, just try to update it here so it always available.

Comment (mfalesni) [reply airgun/navigation.py:49]: We use prerequisite_view to make it visually clear that it is the view from the previous step, which you usually use to navigate to the location the class represents inside the step()

Comment (abalakh) [reply airgun/navigation.py:49]: Not sure about this change then. For me `self.parent` looks visually clearer than `self.prerequisite_view` and doesn't introduce another attribute pointing to the same object. @oshtaier your thoughts? Should we follow the same approach?

**Comment by oshtaier** [comment]
`self.parent` sounds better, but if we need to have that `try...except` block - let's keep it

**Comment by oshtaier** [comment]
btw, we following pep8, so long names are not good

**Comment by mfalesni** [comment]
`self.parent` is not always set by the default implementation. We follow PEP8 as well, but this was the best we could come up with :)

**Comment by abalakh** [comment]
@mfalesni why not set `self.parent` then? Similar to your proposal:
```python
if not hasattr(self, 'parent'):
    setattr(self, 'parent', None)
```

**Comment by mfalesni** [comment]
Actually, we should not need it since you assume the prerequisite always defines some view. So this PR is purely for the name, therefore can be probably closed.

**Comment by mfalesni** [comment]
As per irc discussion

**Pull request closed without merging**